src/legacy/core_plugins/kibana/common/field_formats/types/url.js

```diff
@@ -152,12 +153,12 @@ export function createUrlFormat(FieldFormat) {
               prefix = `${parsedUrl.origin}${parsedUrl.pathname}`;
             }
             // Handle urls like: `/app/kibana` or `/xyz/app/kibana`
-            else if (url.indexOf(parsedUrl.basePath || '/') === 0) {
```

Comment:
Not sure that it's right changes. 

What about: 
![image](https://user-images.githubusercontent.com/20072247/65492426-944e8280-deb9-11e9-955e-4fe4372499bf.png)

Reply:
Changed